Docker Desktop 4.7.1 release notes (#14571)
docker/docker.github.io: desktop/windows/release-notes/index.md

Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.

## Docker Desktop 4.7.1
2022-04-19

> Download Docker Desktop
>
> [For
> Windows](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){:
> .button .primary-btn }

### Bug fixes and minor changes
 - Fixed a bug where update was failing with a symlink error. Fixes [docker/for-win#12650](https://github.com/docker/for-win/issues/12650).

>  - Fixed a bug that prevent starting in windows contianer mode because some layers contained symlinks. Fixes [docker/for-win#12652](https://github.com/docker/for-win/issues/12652).

Reviewer: ```suggestion
 - Fixed a bug that prevented starting Docker Desktop in Windows container mode because some layers contained symlinks. Fixes [docker/for-win#12652](https://github.com/docker/for-win/issues/12652).
```

>  - Fixed a bug that prevent starting in windows contianer mode because some layers contained symlinks. Fixes [docker/for-win#12652](https://github.com/docker/for-win/issues/12652).

Reviewer 2: ```suggestion
 - Fixed a bug that prevented starting in Windows container mode. Fixes [docker/for-win#12652](https://github.com/docker/for-win/issues/12652).
```

Reviewer 2: Also, is it really "starting in" Windows container mode? That would imply that it's OK if you switch to Windows containers later. If you can't use Windows containers at all, it would be better to write "a bug that prevented using Windows container mode".

Author: Yes, it was using windows containers. Even the switch didn't work, I change that.

>  - Fixed crash on the Quick Start Guide final screen.

Reviewer: ```suggestion
 - Fixed a crash on the Quick Start Guide final screen.
```
